x-pack/plugin/src/test/resources/rest-api-spec/test/data_frame/transforms_start_stop.yml

```diff
@@ -141,6 +141,12 @@ teardown:
             "pivot": {
               "group_by": { "airline": {"terms": {"field": "airline"}}},
               "aggs": {"avg_response": {"avg": {"field": "responsetime"}}}
+            },
```

Comment:
I changed this config on purpose. I noticed while writing the `yaml` test for this PR, that this particular test was saying that its transform was continuous, but it actually wasn't. So, I just fixed it.